main.go

```diff
@@ -3,8 +3,9 @@ package main
 import (
 	"os"
 
-	"github.com/GoBootcamp/clirescue/trackerapi"
 	"github.com/codegangsta/cli"
+
+	"./trackerapi"
```

Comment:
here and next not use relative path at import, it works unpredictably at different platforms